tests/python/unittest/test_sparse_operator.py

```diff
@@ -349,6 +349,18 @@ def check_elemwise_binary_ops(lhs_stype, rhs_stype, shape,
                                 lhs_density=lhs_density, rhs_density=rhs_density,
                                 verbose=False)
 
+        if lhs_stype is 'default' and rhs_stype is 'row_sparse':
```

Comment:
is inplace elemwise_sub not tested?
is inplace(dns, csr) not tested?

Reply:
for elemwise_sub, inplace and writeto have the same code(remember the negation step?)
Will add for dns, csr